src/main/scala/api/commands/blastn.scala

```diff
@@ -0,0 +1,85 @@
+package ohnosequences.blast.api
+
+import ohnosequences.cosas._, types._, records._, klists._
+
+// type blastn = blastn.type
+case object blastn extends AnyBlastCommand {
+
+  type Arguments = arguments.type
+  case object arguments extends RecordType(db :×: query :×: out :×: |[AnyBlastOption])
+
+  type Options = options.type
+  case object options extends RecordType(
+    num_threads     :×:
+    task            :×:
+    evalue          :×:
+    max_target_seqs :×:
+    strand          :×:
+    word_size       :×:
+    show_gis        :×:
+    ungapped        :×:
+    |[AnyBlastOption]
+  )
+
+  type ArgumentsVals =
```

Comment:
You already had explicit `OptionVals` type everywhere, so I thought that adding `ArgumentsVals` is not a big loss. We can infer this implicitly, but I don't see much point. This is ugly indeed, but it's simple.

What I would do instead, is having separate constructors for `ArgumentsFor[C <: AnyBlastCommand]` and for options, which would infer this type and make all this `toSeq` juggling unnecessary.